src/main/java/org/embl/mobie/viewer/transform/MergedGridSourceTransformer.java

```diff
@@ -41,6 +40,11 @@ public void transform( Map< String, SourceAndConverter< ? > > sourceNameToSource
 
 		sourceNameToSourceAndConverter.put( mergedSourceAndConverter.getSpimSource().getName(), mergedSourceAndConverter );
 
+		if ( mergedGridSourceName.equals( "plate_cell_segmentation" ) )
+		{
+			int a = 1;
```

Comment:
I guess this hardcoding needs to be removed for 'plate_cell_segmentation'?

Reply:
Good catch! I removed it.